src/lib/TopLevel.hs

```diff
@@ -353,26 +387,35 @@ _evalMLIR block' = do
 _evalMLIR = error "Dex built without support for MLIR"
 #endif
 
+-- TODO: there's no guarantee this name isn't already taken. Need a better story
+-- for C/LLVM function names
+mainFuncName :: SourceName
+mainFuncName = "entryFun"
```

Comment:
This is probably fine, but it would be a good idea to have a prefix we always append to Dex generated function symbols. Something like `__dex__`